canonical-websites/vanillaframework.io
# Update to Vanilla 1.7.0

## Done

- Updated to Vanilla 1.7.0
- Removed bespoke styling

## QA

- Pull code, `./run`
- Check it looks good

## Issue / Card

Fixes #126
